tests/unit/test_adapter.py

```diff
@@ -155,12 +193,12 @@ def test_proteomics_adapter(eciML1515):
         [],
         [],
         proteomics,
-        [],
+        uptake_secretion_rates,
         [],
         growth_rate,
     )
-    # 2 operations (1 proteomics + growth rate) + 3 warnings (1 skipped protein + 2
-    # removed proteins) are expected:
-    assert len(operations) == 2
+    # 2 operations (1 protein + 1 uptake + 1 secretion + growth rate) + 3 warnings
```

Comment:
Typo, should say 4 operations

Reply:
fixed in [`66ac317`](https://github.com/DD-DeCaF/simulations/pull/148/commits/66ac3174720fc62cf01e9ae0e9dd38fcac4171dc)